pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1501,8 +1512,8 @@ def marker_to_center_frequency(self):
     marker_frequency = Instrument.control(
         "MKF?", "MKF %.11E",
         """
-        Place an active marker on the chosen frequency or can be queried to
-        return the frequency of the active marker. Default units are in Hertz.
+        Set an active marker on the chosen frequency or get the frequency of the active marker.
+        Default units are in Hertz.
```

Comment:
What about this:
```suggestion
        Control the frequency of the active marker.
        Default units are in Hertz.
```
"Control" means: get or set it.